node/src/testing/network.rs

```diff
@@ -276,6 +276,23 @@ where
         &self.nodes
     }
 
+    /// Returns the internal map of nodes.
+    pub fn nodes_mut(&mut self) -> &mut HashMap<R::NodeId, Runner<ConditionCheckReactor<R>>> {
+        &mut self.nodes
+    }
```

Comment:
We might as well just make the member variable public

Reply:
I reserve the right to change the underlying data structure in the future :)